base/repl/REPL.jl

```diff
@@ -689,7 +690,11 @@ repl_filename(repl, hp) = "REPL"
 const JL_PROMPT_PASTE = Ref(true)
 enable_promptpaste(v::Bool) = JL_PROMPT_PASTE[] = v
 
-function setup_interface(repl::LineEditREPL; hascolor = repl.hascolor, extra_repl_keymap = Dict{Any,Any}[])
+function setup_interface(
+    repl::LineEditREPL;
+    hascolor::Bool = repl.hascolor,
+    extra_repl_keymap::Vector{<:Dict} = Dict{Any,Any}[]
```

Comment:
Then why not `Vector{Dict}` ? The subtype operator on a concrete type looks confusing